plugins/@grouparoo/spec-helper/src/lib/factories/apiKey.ts

```diff
@@ -14,5 +15,5 @@ const data = async (props = {}) => {
 
 export default async (props = {}) => {
   const { ApiKey } = await import(`@grouparoo/core/${loadPath}`);
-  return ApiKey.create(await data(props));
+  return ApiKey.create(await data(props)) as ApiKey;
```

Comment:
The instance of the object the factory creates is dynamically sourced from the proper location (src or dist), but then the type is applied to the response